bin/filter_bed.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+infile=$1
+outfile=$2
+
+cat $infile | paste -d '\t' - - | sed 's/-/_/g' | awk 'BEGIN {FS="\t"; OFS="\t"} { if ($1 > $7) { print substr($4,1,length($4)-2),$12,$7,$8,"16",$6,$1,$2,"8",$11,$5 } else { print substr($4,1,length($4)-2),$6,$1,$2,"8",$12,$7,$8,"16",$5,$11} }' | tr '_+' '01' | awk 'NF==11' > $outfile
```

Comment:
For increased readability
```suggestion
cat $infile |
    paste -d '\t' - - |
    sed 's/-/_/g' |
    awk 'BEGIN {FS="\t"; OFS="\t"} { if ($1 > $7) { print substr($4,1,length($4)-2),$12,$7,$8,"16",$6,$1,$2,"8",$11,$5 } else { print substr($4,1,length($4)-2),$6,$1,$2,"8",$12,$7,$8,"16",$5,$11} }' |
    tr '_+' '01' |
    awk 'NF==11' > $outfile
```

Reply:
I am going to test this 🤞🏼 I did something similar and kept getting errors at the first `awk` stage.